lib/translator/mode_sense.h

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef LIB_TRANSLATOR_MODE_SENSE_H
+#define LIB_TRANSLATOR_MODE_SENSE_H
+
+#include "common.h"
+
+#include "third_party/spdk/nvme.h"
+
+namespace translator {
+
+StatusCode ModeSense6ToNvme(Span<const uint8_t> scsi_cmd,
```

Comment:
please add comment about what mode sense gets translated into

Reply:
similarly to #134 , links to translation specs are in the .cc, should we also place them in the .h?

Reply:
Follow discussion there, a note will be made in the .h